docs/_cli_docs/docs.md

```diff
@@ -3347,6 +3463,4 @@ You can get help by either emailing `partners@zapier.com` or by [joining our dev
 
 ## Developing on the CLI
 
-See [CONTRIBUTING.md](https://github.com/zapier/zapier-platform/blob/master/CONTRIBUTING.md).
-
-{% endraw %}
```

Comment:
Was this purposely removed? If so, we'll likely want to also want to remove the `{% raw %}` at the beginning of the file as well